src/styles/blocks/card.scss

```
@@ -0,0 +1,79 @@
+.card {
+  margin: 0 auto;
+  width: 200px;
+  background-color: $main-back-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: flex;
+    margin-bottom: 40px;
+    justify-content: center;
+  }
+
+  & .image {
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 150%;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 140%;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-title {
+      font-weight: 400;
+      font-size: 12px;
+      line-height: 150%;
+      color: #616070;
```

```suggestion
      color: $second-text-color;
```
Be attentive, please